src/xenia/kernel/xboxkrnl/xboxkrnl_memory.cc

```diff
@@ -425,8 +425,14 @@ DECLARE_XBOXKRNL_EXPORT2(MmQueryAddressProtect, kMemory, kImplemented,
 
 void MmSetAddressProtect_entry(lpvoid_t base_address, dword_t region_size,
                                dword_t protect_bits) {
-  if (!protect_bits) {
-    XELOGE("MmSetAddressProtect: Failed due to incorrect protect_bits");
+  constexpr uint32_t required_protect_bits =
+      X_PAGE_NOACCESS | X_PAGE_READONLY | X_PAGE_READWRITE |
```

Comment:
What are the exact bits set by those titles that aren't just zero, are those merely something like `X_PAGE_NOCACHE`/`X_PAGE_WRITECOMBINE` purely, or values like `X_PAGE_EXECUTE` also?

Reply:
Titles that I'm aware of (these two in description) use 0 or X_PAGE_NOCACHE, so it's quite difficult to precisely say